src/lib.rs

```diff
@@ -48,23 +48,36 @@
 extern crate capstone_sys;
 extern crate libc;
 
-mod builders;
+pub mod arch;
 mod capstone;
 mod constants;
-mod instruction;
 mod error;
+mod instruction;
 
-pub use builders::*;
 pub use capstone::*;
 pub use constants::*;
 pub use instruction::*;
 pub use error::*;
 
+/// Contains items that you probably want to always import
+///
+/// For example:
+///
+/// ```
+/// use capstone::prelude::*;
+/// ```
+pub mod prelude {
+    pub use {Capstone, CsResult};
+    pub use arch::{self, BuildsCapstone, BuildsCapstoneEndian, BuildsCapstoneExtraMode,
```

Comment:
I don't know how common it is to have a module imported via a prelude, but it makes examples look nicer, so yea